documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -107,9 +121,98 @@ EOF
 <2> Specify the _managed object reference_ of the VMware host.
 <3> Specify the IP address of the VMware migration network.
 
+. If you want to map multiple networks, create a `NetworkAttachmentDefinition` CR manifest for each additional destination network:
++
+[source,yaml]
+----
+$ cat << EOF | oc create -f -
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: <networkattachmentdefinition_name>
+spec:
+  config: '{
+    "cniVersion": "0.4.0",
+    "name": "<networkattachmentdefinition_name>",
+    "plugins": [
+      {
+        "type": "cnv-bridge",
+        "bridge": "br1",
```

Comment:
"br1" is just an example of a bridge name. It can be any name that was configured on that cluster.

Reply:
Maayan said only "br1" worked. Should I say that this should be used if no bridge was configured?

Reply:
On second thought, the CNV doc explains NADs quite well, so I will just link to that section.

Reply:
Here is Petr Horacek's answer regarding  that only "br1" as bridge name was worked for me:
"The bridge you are referring to must be configured on the cluster" (as Ilanit said)
So as I understand only "br1" was configured as "cnv-bridge" bridge on the cluster where I tested NAD creation

Reply:
It doesn't matter because in the end I removed the NAD yaml and replaced it with a link to CNV docs.

Reply:
I saw. I think this better.
Thanks